.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Apart from Bug Report template and feature request, do we need to also have security policy in this repo as present in jx repo?